guides/source/configuring.md

```diff
@@ -1684,6 +1684,14 @@ required:
 config.active_record.database_cli = { postgresql: "pgcli", mysql: %w[ mycli mysql ] }
 ```
 
+#### `config.active_record.use_legacy_signed_id_verifier`
+
+This option allows for a smooth transition during the deprecation of the legacy `signed_id_verifier`. It supports the following modes:
+- `:generate_and_verify` (default) – Generates signed IDs using the legacy format while verifying both the new and legacy formats.
+- `:verify` – Generates signed IDs using the new format while verifying both the new and legacy formats.
```

Comment:
We need to include the "false" option here too.

Reply:
Yes, totally agree. I was also thinking about it. 👍